image_entropy_split.py

```diff
@@ -4,10 +4,14 @@
 from pathlib import Path
```

Comment:
Maybe you could move this file to the folder utils_scripts/ ? :) I think it would make more sense for it to be there.